package.json

```diff
@@ -114,7 +114,8 @@
     "db:validate": "nx run api:prisma:validate",
     "db:format": "nx run api:prisma:format",
     "db:reset": "nx run api:prisma:reset",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "sentry:sourcemaps": "sentry-cli sourcemaps inject ./dist && sentry-cli sourcemaps upload ./dist || echo 'Failed to upload source maps to Sentry'"
```

Comment:
Issue a non-zero exit code in here so that the overall build fails in case something like this happens